app/controllers/contact_tickets_controller.rb

```diff
@@ -7,9 +7,12 @@ class ContactTicketsController < ApplicationController
   def index
     # Don't need the access token for seeing tickets but don't return ticket_id
     # otherwise people can "steal" tickets from other people
-    json_response(Ticket.where(contact: @contact).map do |ticket|
-      ticket.as_json.except(:ticket_id)
-    end)
+    json_response(
+      Ticket.where(contact: @contact)
+        .order(:associated_with_contact_at).map do |ticket|
```

Comment:
Should we specify an order direction? Even if we want the default, it could make it clearer / more readable